train/tr10-13B-ml/tr10-13B.slurm

```diff
@@ -57,13 +57,14 @@ OPTIMIZER_ARGS=" \
     --adam-beta1 0.9 \
     --adam-beta2 0.95 \
     --adam-eps 1e-8 \
-    --lr 6e-5 \
+    --lr 1e-4 \
     --min-lr 6e-6 \
     --lr-decay-style cosine \
-    --lr-decay-samples 126_953_125 \
```

Comment:
you removed this one w/o any commentary?

Reply:
The original tr1-13B said:

> We need lr-decay in samples, so tokens2samples = 260B / 2048 = 126_953_125

Reply:
I was looking at setting it by default to the entire number of samples we have
https://github.com/bigscience-workshop/Megatron-DeepSpeed/blob/fd1e1da967c74e598acfc011031474663ef5845e/megatron/training.py#L341

We have been using this in arch/scaling. 

However I've just re-read the GPT3 paper and they do it for 260B ... so not sure here. cc @TevenLeScao

Reply:
Thank you for the note, Thomas - it's crucial that we leave a note trail, otherwise we have no idea why some config was added or removed.